subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/core_dependency_management.adoc

```diff
@@ -14,36 +15,52 @@ For an overview of dependency management terms, see <<dependency_management_term
 [[sec:dependency-mgmt-in-gradle]]
 == Dependency Management in Gradle
 
-[.inset]
+.Dependencies management at a glance
 image::dependency-management-resolution.png[]
 
 Gradle has built-in support for dependency management.
 
-We’ll explore the main concepts with the help of the theoretical but common project to define terms:
+Let's explore the main concepts with the help of a theoretical but common project:
 
 * This project builds Java source code.
-* Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library.
+* Some Java source files import classes from the link:https://github.com/google/guava[Google Guava] library.
 * This project uses link:http://junit.org/junit5/[JUnit] for testing.
 
-This project uses Guava and JUnit as _dependencies_.
-Gradle fetches dependencies from _repositories_.
-You can <<declaring_repositories.adoc#declaring-repositories,declare repositories>> to tell Gradle where to fetch dependencies.
+The `build.gradle` file might look as follows:
+
+.Gradle build file with dependencies
+====
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/groovy",files="build.gradle"]
+====
+<1> Define artifact sources for project
```

Comment:
The terminology used in the call outs does not match the rest of the doc.

Given that you effectively explain the different pieces below, I would suggest to:
* Either move the sentences below to the right callout
* Or remove the callouts altogether and rely on the paragraphs below to explain the snippet.

Reply:
> Either move the sentences below to the right callout
Chose this in the latest commit.